bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -1607,6 +1607,73 @@
           expect(vendor_files).to include("statesman-7.2.0.gem")
         end
       end
+
+      context "with a git dependency" do
+        let(:repo_contents_path) { build_tmp_repo("vendored_git") }
+        let(:gemfile_fixture_name) { "git_source_with_version" }
+        let(:lockfile_fixture_name) { "git_source_with_version.lock" }
```

Comment:
In #2476 I've since restructured this to use a Gemfile and Gemfile.lock in the project path, would be great to do the same here

Reply:
Awesomesauce! That disconnect [threw me for a loop](https://github.com/dependabot/dependabot-core/pull/2476#discussion_r476693529) until I found the indirection.